packages/manager/modules/pci/src/projects/project/training/onboarding/onboarding.html

```diff
@@ -14,9 +14,9 @@ <h1 data-translate="pci_projects_project_training_title"></h1>
         data-on-accept="$ctrl.acceptLab(accepted)"
     ></pci-project-lab-agreements>
 
-    <span data-ng-if="$ctrl.error">
+    <p class="text-danger" data-ng-if="$ctrl.error">
```

Comment:
Maybe we can consider using a [`<oui-message type="error">`](https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-message-webcomponent--default).

Does that make sense for you?

Reply:
Replaced `<p>` by `<oui-message type="error">`